resolwe_bio/docker_images/dnaseq/README.md

```diff
@@ -5,11 +5,11 @@ https://hub.docker.com/r/resolwebio/base/) image.
 
 Included bioinformatics tools:
 ------------------------------
-* bedtools (v2.26.0+dfsg-3)
-* bwa (v0.7.15-3)
+* bedtools (v2.26.0+dfsg-5)
+* bwa (v0.7.17-1)
 * CheMut
-* Picard Tools (v2.8.1+dfsg-2)
+* Picard Tools (v2.8.1+dfsg-3)
 * Primerclip (v171018)
-* Rsamtools (v1.26.2)
+* Rsamtools (v1.30.0)
 * SAMtools (v1.6)
```

Comment:
Consider updating SAMtools. The version installed here manually is older than the version that comes packaged with Ubuntu 18.04.

Reply:
Update the version here as well.

Reply:
Done.